src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1840,6 +1882,94 @@
       "required": [ "target", "kinds" ]
     },
 
+    "request": {
+      "description": "A web request object.",
+      "type": "object",
+      "properties": {
+
+        "index": {
+          "description": "The index within the run.requests array of the request object associated with this result.",
+          "type": "integer"
+        },
+
+        "uri": {
+          "description": "The uri associated with the web request.",
+          "type": "string",
+          "format": "uri"
+        },
+
+        "verb": {
+          "description": "The verb associated with the web request. Well-known values include GET, POST, PUT, DELETE, PATCH.",
+          "type": "string"
+        },
+
+        "headers": {
+          "description": "The headers associated with the web request.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+
+        "parameters": {
+          "description": "The parameters associated with the web request.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+
+        "body": {
+          "description": "The body associated with the web request.",
+          "type": "string"
+        },
+
+        "properties": {
+          "description": "Key/value pairs that provide additional information about the request.",
+          "$ref": "#/definitions/propertyBag"
+        }
+      },
+      "required": [ "verb" ],
+      "anyOf": [
+        {
+          "required": [ "index" ]
+        },
+        {
+          "required": [ "uri", "index"]
```

Comment:
>index [](start = 32, length = 5)

This "index" shouldn't be necessary.

Reply:
oops! thanks!

---
In reply to: [273674746](https://github.com/Microsoft/sarif-sdk/pull/1419#discussion_r273674746) [](ancestors = 273674746)

Reply:
marking resolved since we decided to remove these conditions from schema.

---
In reply to: [273691495](https://github.com/Microsoft/sarif-sdk/pull/1419#discussion_r273691495) [](ancestors = 273691495,273674746)